### PR TITLE
Show quorum connections in "quorum dkgstatus" and use it in mine_quorum

### PR DESCRIPTION
**Pull request by codablock**
Instead of statically sleeping 2 seconds

### CODE REVIEW

File: test/functional/test_framework/test_framework.py [hunk 1]
```diff
@@ -740,6 +740,30 @@ def check_sporks_same():
...
+            all_ok = True
+            for node in self.nodes:
+                s = node.quorum("dkgstatus")
+                if s["session"] == {}:
+                    continue
+                if "quorumConnections" not in s:
+                    all_ok = False
+                    break
+                s = s["quorumConnections"]
+                if "llmq_test" not in s:
```
Comment (UdjinM6) [inline test/functional/test_framework/test_framework.py]: I guess this needs #3269 to be merged first

File: test/functional/test_framework/test_framework.py [hunk 2]
```diff
@@ -740,6 +740,30 @@ def check_sporks_same():
...
+                    all_ok = False
+                    break
+                cnt = 0
+                for c in s["llmq_test"]:
```
Comment (UdjinM6) [inline test/functional/test_framework/test_framework.py]: same

File: test/functional/test_framework/test_framework.py (hunk 1 above)
Comment (codablock) [reply test/functional/test_framework/test_framework.py]: oh..yepp..this PR came out of one big branch and I didn't notice this depenency. I'll rebase this later

**Comment by codablock** [comment]
rebased on develop after merge of https://github.com/dashpay/dash/pull/3269. This should fix tests

**Comment by codablock** [comment]
Test failures on Travis are unrelated

**Pull request merged**